app/views/index.njk

```diff
@@ -45,7 +45,7 @@
 
       {% if not applications.length %}
 
-        <p class="govuk-body">No applications found.</p>
+        <p class="govuk-body">No results for ‘{{ data.keywords }}’ {%- if hasFilters %} and the selected filter{% if selectedFilters.categories[0].items.length > 1 %}s{% endif %}{% endif %}.</p>
```

Comment:
```suggestion
        <p class="govuk-body">There are no results for ‘{{ data.keywords }}’ {%- if hasFilters %} with the selected filter{% if selectedFilters.categories[0].items.length > 1 %}s{% endif %}{% endif %}.</p>
```

I was going to suggest removing the full stop as it wasn't a complete sentence. But actually I think it's worth making it a complete sentence by adding "There are" at the start. And I'd prefer "with the" rather than "and the".

Reply:
Okay change made.